### ISSUE TITLE
Applications cannot be opened from Application menu on KDE

### ISSUE THREAD

**Issue by kaanchinar** [winapps-org/winapps#89]
I freshly installed and configured my win10 VM with winapps. All programs work fine via terminal.
```bash
winapps word-o365-x86
```
Hovewer, when I attempt to start same program from Application menu, I appears on my taskbar for a second, then disappears without any screen and error.

How can I solve that issue?

Using Arch Linux with FreeRDP 2 and KDE Plasma 6

**Comment by oskardotglobal** [comment]
Did you try manually checking what command the desktop files attempt to run? Maybe it's an issue with the path to the executable?

**Comment by kaanchinar** [comment]
> Did you try manually checking what command the desktop files attempt to run? Maybe it's an issue with the path to the executable?

I did. Changed `chmod`, edited .desktop entries with different combinations (with sudo command, direct path etc.) but no success.

**Comment by oskardotglobal** [comment]
Can you try this again with the latest commit?

**Comment by oskardotglobal** [comment]
Also, please upgrade to freerdp3

**Comment by oskardotglobal** [comment]
Moving this to #104

**Comment by Sithuk** [comment]
I am experiencing the same issue with a fresh install from the master. I'm on Arch. The Exec command line works perfectly from the terminal. The menu item fails silently.

**Comment by Sithuk** [comment]
I've tracked the issue down to the waCheckVMRunning function by uncommenting "set -x" in the winapps script and checking journalctl. 

virsh is not returning the running VM name from the following command.

virsh list --state-running --name

I've added --debug 1 to the command in the script and logged to a file to confirm that the VM name isn't being returned.

virsh --debug 1 list --state-running --name

When I run the "virsh list --state-running --name" command at the terminal, it returns the VM name. 

I've put "whoami" in the function and confirmed that the script is being run as my user when activated from the menu.

I have freeRDP3.
